airbyte-integrations/connectors/source-amplitude/source_amplitude/api.py

```diff
@@ -69,6 +69,27 @@ def time_interval(self) -> dict:
         """
         pass
 
+    def _get_date_time_items_from_schema(self):
+        """
+        Get all properties from schema with format: 'date-time'
+        """
+        result = []
+        schema = self.get_json_schema()["properties"]
+        for key, value in schema.items():
+            if value.get("format") == "date-time":
+                result.append(key)
+        return result
+
```

Comment:
Shouldn't this functionality maybe be added at the CDK level in the type transformers?

Reply:
Great idea, i'll make another issue for this)